### PR TITLE
members modal notices

### PR DESCRIPTION
**Pull request by razvanaldea89**

### CODE REVIEW

File: src/MembersLock/classes/class-dlm-members-modal.php [hunk 1]
```diff
@@ -255,9 +255,25 @@ public function login_user() {
...
 		$info['remember']      = true;
 		$user_sigon            = wp_signon( $info, false );
+		$download              = false;
+		$download_link         = '';
+		// Get the Download
+		if ( ! empty( $_POST['download_id'] ) ) {
+			// Get the Download
+			try {
+				$download = download_monitor()->service( 'download_repository' )->retrieve_single( absint( $_POST['download_id'] ) );
+			} catch ( Exception $e ) {
```
Comment (TeoAlex) [inline src/MembersLock/classes/class-dlm-members-modal.php:265]: trimite un json error?

File: src/MembersLock/classes/class-dlm-members-modal.php [hunk 2]
```diff
@@ -255,9 +255,25 @@ public function login_user() {
...
+			}
+		}
+		// Check if user managed to log in
 		if ( is_wp_error( $user_sigon ) ) {
 			wp_send_json_error( $user_sigon->get_error_message() );
 		}
-		wp_send_json_success( 'success' );
+		// Found Download? Set download link
+		if ( $download ) {
+			$download_link = '<p>' . esc_html__( 'Authentification succesfull. Download link: ', 'download-monitor' ) . '<a href="' . esc_url( $download->get_the_download_link() ) . '">' . esc_html( $download->get_title() ) . '</a><p>';
```
Comment (TeoAlex) [inline src/MembersLock/classes/class-dlm-members-modal.php:274]: successful

**Review by TeoAlex — CHANGES_REQUESTED** [review]

File: src/MembersLock/classes/class-dlm-members-modal.php (hunk 1 above)
Comment (razvanaldea89) [reply src/MembersLock/classes/class-dlm-members-modal.php:265]: Nup, pur si simplu lasam omul sa se autentifice dar nu mai trimitem link-ul de download.

File: src/MembersLock/classes/class-dlm-members-modal.php (hunk 2 above)
Comment (razvanaldea89) [reply src/MembersLock/classes/class-dlm-members-modal.php:274]: 88575fb1c8bde7ac73a2311af579f013d619d5fa commit acilea

Comment (razvanaldea89) [reply src/MembersLock/classes/class-dlm-members-modal.php:274]: 88575fb1c8bde7ac73a2311af579f013d619d5fa

**Review by TeoAlex — APPROVED** [review]

**Pull request merged**